dragonchain/lib/dto/schema.py

```diff
@@ -669,3 +669,49 @@ def get_transaction_queue_task_schema(dict_payload: bool = False) -> Dict[str, A
     "required": ["to", "value"],
     "additionalProperties": False,
 }
+
+# BINANCE INTERCHAIN #
+
+create_binance_interchain_schema_v1 = {
+    "type": "object",
+    "properties": {
+        "version": {"type": "string", "enum": ["1"]},
+        "name": {"type": "string"},
+        "testnet": {"type": "boolean"},
+        "node_url": {"type": "string"},
+        "rpc_port": {"type": "integer"},
+        "api_port": {"type": "integer"},
+        "private_key": {"type": "string"},
+    },
+    "additionalProperties": False,
+    "required": ["name", "version"],
+}
+
+# Same as create without required name field
+update_binance_interchain_schema_v1 = {
+    "type": "object",
+    "properties": {
+        "version": {"type": "string", "enum": ["1"]},
+        "name": {"type": "string"},
+        "testnet": {"type": "boolean"},
+        "node_url": {"type": "string"},
+        "rpc_port": {"type": "integer"},
+        "api_port": {"type": "integer"},
+        "private_key": {"type": "string"},
+    },
+    "additionalProperties": False,
+    "required": ["version"],
+}
+
+bnb_transaction_schema_v1 = {
+    "type": "object",
+    "properties": {
+        "version": {"type": "string", "enum": ["1"]},
+        "symbol": {"type": "string"},
+        "amount": {"type": "number"},
+        "to_address": {"type": "string"},
+        "memo": {"type": "string"},
+    },
+    "required": ["amount", "to_address"],
```

Comment:
version should be required

Reply:
Version is not required in BTC and ETH transaction schemas.

Reply:
Done.